lib/init.coffee

```diff
@@ -78,127 +81,76 @@ module.exports =
       helpers = require 'atom-linter'
       voucher = require 'voucher'
       fs = require 'fs'
-      if @verboseLogging
-        @_log 'requiring modules finished.'
+      @_log 'requiring modules finished.'
 
-    if @verboseLogging
-      @_log 'providing linter, examining javac-callability.'
+    @_log 'providing linter, examining javac-callability.'
 
     grammarScopes: ['source.java']
     scope: 'project'
     lintOnFly: false       # Only lint on save
+
     lint: (textEditor) =>
       filePath = textEditor.getPath()
       wd = path.dirname filePath
       searchDir = @getProjectRootDir() || path.dirname filePath
+
       # Classpath
       cp = ''
 
-      if @verboseLogging
-        @_log 'starting to lint.'
+      # Sourcepath - this is a fair default for many Java projects
+      sp = 'src/main/java/'
```

Comment:
I agree to that, i guess i just didn't express well. I would prefer to put `src/main/java/` into the config-setting as default. This would avoid that the user might think no default is set (since i guess some people just skip the fine-prints in the settings-section).